test/e2e/cypress/support/config.d.ts

```diff
@@ -0,0 +1,1185 @@
+// Copyright © 2023 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+/* eslint-disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+export type OryKratosConfiguration = OryKratosConfiguration1 &
+  OryKratosConfiguration2
+export type OryKratosConfiguration1 = {
+  [k: string]: unknown | undefined
+}
```

Comment:
Not sure, why it generates these 1, 2 versions, but I traced it to being related to the `allOf` block at the end of the config. Not sure if there is anything we can do about that here.